.travis.yml

```diff
@@ -151,7 +151,7 @@ env:
     PARENT_GROUP=all
     HOST_GROUP=db_server
     PLAYBOOK_LOCATION=/aurora/deploy/deploy_db_server.yml
-    EXTRA_VAR_DEF='{"deploy_iptables": false, "install_ppas": false, "skip_glusterfs": true}'
+    EXTRA_VAR_DEF='{"deploy_iptables": false, "skip_glusterfs": true}'
```

Comment:
@rioh Can you add `pp_install: true`?  This is required in order to cause the role to run the `initdb` task, without that Travis will fail.  Hopefully this will finally allow Travis to run through completely to verify our Postgres scripts!